translate/storage/base.py

```diff
@@ -408,6 +408,9 @@ def istranslatable(self):
         """
         return bool(self.source)
 
+    def marktranslateable(self, value):
```

Comment:
As the getter is called `istranslatble`, this should be named consistently. Also I'd prefer it to accept boolean in the API as the string values are Android-specific:

```suggestion
    def marktranslatable(self, value: bool) -> None:
```

Reply:
Updated the code to this change suggestion.